libs/dyn/path.go

```diff
@@ -18,11 +18,11 @@ func (c pathComponent) Index() int {
 	return c.index
 }
 
-func (c pathComponent) isKey() bool {
+func (c pathComponent) IsKey() bool {
 	return c.key != ""
 }
 
-func (c pathComponent) isIndex() bool {
+func (c pathComponent) IsIndex() bool {
```

Comment:
I don't see these being used outside the pkg (in this commit).

Reply:
thanks for catching, indeed not used anymore